src/explorer/src/pages/index.astro

```diff
@@ -10,35 +10,49 @@ import { Image } from 'astro:assets';
 ---
 
 <Chrome>
-  <section class="bg-slate-100 bg-hero-topography-wavy-10 p-6 border-b border-slate-400 shadow-xs">
-    <div class="text-center mx-auto max-w-3xl">
-      <h1 class="md:text-5xl text-3xl text-slate-800 font-extrabold tracking-tight mt-4 mb-0">
+  <section
+    class="bg-hero-topography-wavy-10 shadow-xs border-b border-slate-400 bg-slate-100 p-6"
+  >
+    <div class="mx-auto max-w-3xl text-center">
+      <h1
+        class="mb-0 mt-4 text-3xl font-extrabold tracking-tight text-slate-800 md:text-5xl"
+      >
         Data to enrich your <span class="text-wavy-600">business</span>
       </h1>
-      <h2 class="md:text-3xl text-2xl block text-slate-700 font-light tracking-tight m-0">
+      <h2
+        class="m-0 block text-2xl font-light tracking-tight text-slate-700 md:text-3xl"
+      >
         An API to access <span class="font-semibold">Utah</span> geospatial data.
       </h2>
-      <p class="md:text-lg text-base mt-3 font-normal text-slate-700 leading-relaxed tracking-wide">
-        The Utah Geospatial Resource Center (UGRC) is the State of Utah's map technology coordination office. The UGRC
-        creates, maintains, and stores geospatial data in the State Geographic Information Database (SGID), a one-stop
-        shop to hundreds of data layers developed, aggregated, or acquired by state government. This web API gives you
-        completely open access to
-        <span class="text-2xl block text-wavy-600 font-bold tracking-tighter"> all of that data.</span>
+      <p
+        class="mt-3 text-base font-normal leading-relaxed tracking-wide text-slate-700 md:text-lg"
+      >
+        The Utah Geospatial Resource Center (UGRC) is the State of Utah's map
+        technology coordination office. The UGRC creates, maintains, and stores
+        geospatial data in the State Geographic Information Database (SGID), a
+        one-stop shop to hundreds of data layers developed, aggregated, or
+        acquired by state government. This web API gives you completely open
+        access to
+        <span class="text-wavy-600 block text-2xl font-bold tracking-tighter">
+          all of that data.</span
+        >
       </p>
       <div class="flex justify-center md:my-4">
         <div
-          class="max-w-xl w-full mt-6 md:mb-6 md:mt-10 flex sm:item-center justify-evenly flex-wrap bg-white p-5 rounded-lg border border-slate-300"
+          class="sm:item-center mt-6 flex w-full max-w-xl flex-wrap justify-evenly rounded-lg border border-slate-300 bg-white p-5 md:mb-6 md:mt-10"
```

Comment:
this will trigger the sm: to be placed at the end...
```suggestion
          class="sm:items-center mt-6 flex w-full max-w-xl flex-wrap justify-evenly rounded-lg border border-slate-300 bg-white p-5 md:mb-6 md:mt-10"
```